project/plugins.sbt

```diff
@@ -29,6 +29,8 @@ addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.3.1")
 
 addSbtPlugin("com.cavorite" % "sbt-avro" % "0.3.2")
 
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.7.2")
```

Comment:
no, this is definitely not necessary.  It was extremely useful to me during development, and figured it was a good addition in general.

This plugin lets you easily start & stop long-running processes (like the history server which never dies) within an sbt session, without having to kill & restart sbt.  You just run `re-start` to launch your main program, but it gives you the sbt shell back, so you can run more commands, and eventually run `re-stop`.

I figured it was harmless to drop in here as well, but I'll move to a separate PR.